### DISCUSSION TITLE
application cannot be reviewed until fixing profile

### DISCUSSION THREAD

**Discussion by Pushistickk** [community/community#34723] [GitHub Education]
"You are significantly more likely to be verified if you have completed your Github user profile with your full name and short bio"
completed my profile, verified email and still not working. Tried every "solution" posted about this problem, but still nothing

**Comment by K0I0S-Codes** [comment]
I have this issue too, and I've tried changing my name, adding information on my bio, even though I already had a bio, and even created a Readme on my profile for extra info, and it just won't take it giving the same message stated below
![image_2022-11-11_134218929](https://user-images.githubusercontent.com/23627450/201409302-16365d7b-df2f-4857-9fef-671dbdb5b218.png)

**Reply by max0n4ik** [reply]
> I have this issue too, and I've tried changing my name, adding information on my bio, even though I already had a bio, and even created a Readme on my profile for extra info, and it just won't take it giving the same message stated below ![image_2022-11-11_134218929](https://user-images.githubusercontent.com/23627450/201409302-16365d7b-df2f-4857-9fef-671dbdb5b218.png)

you find solution?
